crossing-training.lic

```diff
@@ -954,6 +984,17 @@ class CrossingTraining
       false
     end
   end
+  
+  def shape(chapter, unique_name, item)
+    return unless buy_wood
+
+    walk_to(@training_room)
```

Comment:
Should this instead do something similar to https://github.com/rpherbig/dr-scripts/blob/master/workorders.lic#L237 ?